### PR TITLE
SVG and JSON files support

### PR DESCRIPTION
**Pull request by smidhonza**
Added support for loading SVG and JSON files into the preview

### CODE REVIEW

**Comment by romidane** [comment]
Can you update the package.json version too

**Comment by smidhonza** [comment]
yep, updated

**Comment by joshnesbitt** [comment]
:shipit:

File: src/preview-server/bootstrap.js [hunk 1]
```diff
@@ -94,6 +102,10 @@ app.use(
 app.set('view engine', 'ejs');
 app.set('views', `${BLACKJACK_HOME}/src/preview-server/views`);
 
+
+// Serve static files if requested such as images, css, etc.
+app.use('/', express.static('./'));
```
Comment (joshnesbitt) [inline src/preview-server/bootstrap.js]: Should we specify a `public` folder for this?

Comment (joshnesbitt) [inline src/preview-server/bootstrap.js]: Discussed, it's project specific so the root path is fine.

**Pull request merged**